### PR TITLE
[FS-51] Fix Swagger for failed_to_send in Proteus

### PR DESCRIPTION
**Pull request by mdimjasevic**
The response to the `POST /conversations/:domain/:cnv/proteus/messages` endpoint contains the `failed_to_send` field, but the rendered Swagger documentation for the filed was overwritten by another field's documentation. This PR fixes the documentation for that and other response fields listing clients.

Tracked by https://wearezeta.atlassian.net/browse/FS-51.

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

**Comment by mdimjasevic** [comment]
@lepsa , I wanted to add you as a reviewer, but the GitHub interface wouldn't let me. Maybe this has to do with your overall permissions. :man_shrugging:

File: libs/wire-api/src/Wire/API/Message.hs [hunk 1]
```diff
@@ -531,30 +531,39 @@ instance ToSchema MessageSendingStatus where
...
+            "When message sending fails for some clients but succeeds for others, \
+            \e.g., because a remote backend is unreachable, \
+            \this field will contain the list of clients for which the message sending \
+            \failed. This list should be empty when message sending is not even tried, \
+            \like when some clients are missing."
+    where
+      -- This is to ensure that the Swagger model has a unique name, thereby
+      -- allowing for a description specific to that field instead of having it
+      -- overwritten by other models of the same type.
+      qualifiedUserClientsFieldWithDesc name desc =
```
Comment (lepsa) [inline libs/wire-api/src/Wire/API/Message.hs:557]: Should this be generalised out of this function? I'm sure that it will have more uses than just this.

**Comment by lepsa** [comment]
> 

This change looks good to me. Is the swagger output available anywhere?

**Comment by mdimjasevic** [comment]
> > 
> 
> This change looks good to me. Is the swagger output available anywhere?

Sorry, should have included it from the beginning. Here it is:

proteus-message-sending-status-

**Comment by lepsa** [comment]
> > > 
> > 
> > 
> > This change looks good to me. Is the swagger output available anywhere?
> 
> Sorry, should have included it from the beginning. Here it is:
> 
> proteus-message-sending-status-

Looks good to me!

**Comment by mdimjasevic** [comment]
@akshaymankar , does this updated version of the documentation look OK to you?


proteus-message-sending-status-2

**Review by akshaymankar — APPROVED** [review]
Looks great, hopefully we don't have to do this often.

**Pull request merged**